### ISSUE TITLE
cannot be build under cargo

### ISSUE THREAD

**Issue by crockeo** [envoyproxy/envoy-mobile#1237]
i'm writing [envoy-mobile-sys](https:/github.com/crockeo/envoy-mobile-sys) and [envoy-mobile-rs](https://github.com/envoy-mobile-rs) as hobby projects, in addition to working on py-envoy-mobile. [cargo](https://github.com/rust-lang/cargo) doesn't include cargo-based subpackages while attempting to package + publish.

envoy-mobile depends currently on [envoy a55ae7ee8](https://github.com/envoyproxy/envoy/tree/a55ae7ee8b83b64be90ae788e624efcebe29ed5d), which has an explicit `bazel/external/cargo` dir, which is [ignored while building under cargo](https://github.com/rust-lang/cargo/blob/89dcb2aec259b7baeb6aac7acc9f97595932941f/src/cargo/sources/path.rs#L403-L406). 

any commit after [envoy 9aa5c75d6](https://github.com/envoyproxy/envoy/tree/9aa5c75d6be62cda62a1e00a12b681a8377db9ca) can be built just fine, as it removes the dependency on a pre-existing cargo dir.

i'm opening this ticket to track fast-forwarding past 9aa5c75d6, which i'm happy to work on to continue to unblock my downstream work

**Comment by crockeo** [comment]
...this is the part where i slink away in shame, because i now see that #1230 was merged two days ago.

closing this issue as it's no longer relevant.
